testing/jenkins_remote_access.py

```diff
@@ -266,6 +318,26 @@ def change_mesos_creds(mesos_username, service_name):
         ),
         service_name)
 
+def install_datadog_plugin(service_name):
+    ret = make_post(
+        Template(INSTALL_DATADOG).substitute(
+            {
+            }
+        ),
+        service_name)
+    # we're restarting Jenkins, so we sleep after the post to make sure Jenkins is up.
+    time.sleep(60)
```

Comment:
could we instead poll jenkins till it returns a proper response? sleep 60 may introduce some flakiness if jenkins wasnt ready in that timeframe